itou/www/itou_staff_views/urls.py

```diff
@@ -19,4 +19,6 @@
         views.merge_users_confirm,
         name="merge_users_confirm",
     ),
+    path("otp/setup", views.otp_setup, name="otp_setup"),
+    path("otp/confirm/<int:device_id>", views.otp_confirm, name="otp_confirm"),
```

Comment:
Pour distinguer de `confirm-otp`.

```suggestion
    path("otp/confirm/<int:device_id>", views.otp_confirm, name="otp_confirm_device"),
```